File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int i =0, j = numbers.size()-1;
        for (int k=0;k<numbers.size();k++){
            if (numbers[i]+numbers[j]<target){i+=1;}
            else if (numbers[i]+numbers[j]>target){j-=1;}
            else{ return {i+1, j+1};}
        }
        return {};
    }
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i, j = 0, len(numbers)-1
        for k in range(len(numbers)):
            if (numbers[i]+numbers[j])<target: i+=1
            elif (numbers[i]+numbers[j])>target: j-=1
            else: return [i+1, j+1]
